unified-installer/astra-unified-installer.sh

```diff
@@ -2255,6 +2372,15 @@ step_collect_existing_trident_info() {
         logdebug "trident ACP image: not found"
     fi
 
+    # TORC imagePullSecrets
+    local -r torc_pull_secrets="$(echo "$torc_json" | jq -r '.spec.imagePullSecrets | join("\n")' 2> /dev/null)"
+    if [ -n "$torc_pull_secrets" ] && [ "$torc_pull_secrets" != "null" ]; then
+        logdebug "torc pull secrets:${__NEWLINE}$torc_pull_secrets"
+        _EXISTING_TORC_PULL_SECRETS="$torc_pull_secrets"
+    else
+        logdebug "torc pull secrets: not found"
+    fi
```

Comment:
Get the current TORC pull secrets (if any) so we can determine whether we need to add our pull secret to the list or not.